src/util/saveArticles.ts

```diff
@@ -695,7 +695,7 @@ export async function treatMedias(parsoidDoc: DominoElement, mw: MediaWiki, dump
 
     /* Improve image frames */
     const figures = parsoidDoc.getElementsByTagName('figure');
-    const spans = parsoidDoc.querySelectorAll('span[typeof=mw:Image/Frameless]');
+    const spans = parsoidDoc.querySelectorAll('span[typeof~=mw:Image/Frameless],span[typeof~=mw:File/Frameless]');
```

Comment:
Whoops, "span[typeof~=mw:File/Frameless]" is going to match more than just images

Reply:
Then again, `figures` would anyways